### ISSUE TITLE
Qustion about the inference similarity threshold β

### ISSUE THREAD

**Issue by YouthBL** [MMorafah/FLIS#7]
Hello,  thanks a lot for the open source about clustering federated learning and it's great work. When I read the paper, I had a question about the inference similarity threshold β, I want to ask if the β refers to the 'cluster_alpha' or other hyperparameter in the code.

**Comment by MMorafah** [comment]
Hello, Thanks for your interest in our work. It refers to cluster_alpha.

Regards,
Mahdi

**Comment by YouthBL** [comment]
Thanks for your reply. Recently I want to reproduce the result of Table 2. I only change the --beta from 0.5 to 0.1 in 'flis_dc.sh' to run the Dir(0.1) on the Cifar-10 dataset. However, the result I get is poor and far from yours. I want to ask for your help to improve my result and adjust my setting. Maybe I miss some hyperparameter settings. The result of mine is shown as follows.
![屏幕截图 2024-06-07 121339](https://github.com/MMorafah/FLIS/assets/70211060/aa85c10e-4531-4dbf-b42d-f04f47aa0637)

**Comment by MMorafah** [comment]
Hello,

The details of hyper-parameters are discussed in the paper. Please make sure you are using the correct hyper-parameters. The cluster_threshold may vary depending on the random seed you use. I suggest fixating the clients partitioning and random seed so you can find the best cluster_threshold for that specific client partitioning. Hope this helps.

Regards,
Mahdi
